RUMM-883 Add support for 3rd party networking libraries (including Alamofire) (#340)
DataDog/dd-sdk-ios: README.md

![Datadog iOS Log Collection](docs/images/tracing.png)

## Integrations

### Alamofire

> If you use [Alamofire](https://github.com/Alamofire/Alamofire), take a look at [`DatadogAlamofireExtension` library](Sources/DatadogExtensions/Alamofire/) to learn how to auto instrument your requests with Datadog SDK.

Reviewer: ```suggestion
If you use [Alamofire](https://github.com/Alamofire/Alamofire), review the [`DatadogAlamofireExtension` library](Sources/DatadogExtensions/Alamofire/) to learn how to automatically instrument requests with Datadog SDK.
```
